paho/src/main/java/org/eclipse/paho/client/mqttv3/internal/Token.java

```diff
@@ -432,4 +433,18 @@ public String toString()
 		return tok.toString();
 	}
 
+	public MqttWireMessage getResponse() {
+		return response;
+	}
+
+	public int[] getGrantedQos() {
```

Comment:
i think we can avoid this here. We are exposing wiremessage. We can utilise that